gsplat/cuda/csrc/rasterize_to_pixels_fwd.cu

```diff
@@ -53,6 +57,15 @@ __global__ void rasterize_to_pixels_fwd_kernel(
     bool inside = (i < image_height && j < image_width);
     bool done = !inside;
 
+    // when the mask is provided, render 0.0 and return if this tile
```

Comment:
I think this conflicts with when `backgrounds` is provided. In the case of a background is provided, user might expect the background color to be rendered at masked region.

Reply:
Agree! Though it's not required in Grendel, rendering the background color at masked region is generally useful in many other cases to improve computational efficiency. Will fix this.

Reply:
fixed